source/server/moniker_stream_processor.h

```diff
@@ -0,0 +1,11 @@
+#ifndef MONIKER_STREAM_PROCESSOR_H
```

Comment:
If you change the configuration name from `moniker_stream_processor` to `processor_affinity`, everything here should including file name should change to `processor_affinity` or even better `processor_affinity_configuration`

Reply:
Keeping it same as discussed in chat, refer to the Alex's comment above.